commands/src/api/mod.rs

```diff
@@ -0,0 +1,71 @@
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+use uuid::Uuid;
+
+pub mod deployment;
+pub mod webrtc;
+
+/// Error types
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("Failed to deserialize command")]
+    DeserializeCommand,
+}
+
+/// API request packet type
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Request {
+    /// Request body
+    pub body: Body,
+    /// Response correlation string
+    pub respond_to: Option<String>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+/// API request body
+pub enum Body {
+    /// Start pipeline deployment
+    StartDeployment(deployment::StartDeployment),
+    /// Stop deployment by id
+    StopDeployment(uuid::Uuid),
```

Comment:
I know it is a bit more code, but for consistency maybe it is better to introduce `deployment::StopDeployment`

Reply:
Agree. This will help with protocol extensions in the future.